docs/source/development/widget.md

```diff
@@ -251,10 +250,215 @@ See [Storybook](https://6.docs.plone.org/storybook) with available widgets.
+                type: 'email',
+                widget: 'email',
+            },
+            subscribe_to_newsletter: {
+                title: 'Subscribe to newsletter',
+                type: 'boolean',
+                default: false,
+            },
+        },
+        required: ['fullname', 'from',],
+    }}
+    onSubmit={onSubmit}
+    resetOnCancel={true}
+/>
+```
+
+
+
+### If you want to register the widget for arbitrary fields instead or additionally:
+
+```{code-block} jsx
+:emphasize-lines: 5
+import { ToggleWidget } from './ToggleWidget';
+
+const applyConfig = (config) => {
+    // …
+    config.widgets.widget.toggle_widget = ToggleWidget;
+    // …
+    return config;
+};
+export default applyConfig;
+```
+
+This widget named 'toggle_widget' can now be used as follows:
+
+
+```{code-block} jsx
+:emphasize-lines: 27-28
+<Form
+    schema={{
+        fieldsets: [
+            {
+                id: 'default',
+                title: 'Default',
+                fields: [
+                    'fullname',
+                    'from',
+                    'subscribe_to_newsletter',
+                ],
+            },
+        ],
+        properties: {
+            fullname: {
+                title: intl.formatMessage(messages.fullnameTitle),
+                placeholder: intl.formatMessage(messages.fullnameTitle),
+            },
+            from: {
+                title: intl.formatMessage(messages.emailTitle),
+                placeholder: intl.formatMessage(messages.emailTitle),
+                type: 'email',
+                widget: 'email',
+            },
+            subscribe_to_newsletter: {
+                title: 'Subscribe to newsletter',
+                type: 'pipapo',
+                widget: toggle_widget
+                default: false,
+            },
+        },
+        required: ['fullname', 'from',],
+    }}
+    onSubmit={onSubmit}
+    resetOnCancel={true}
+/>
+```
+
+### Pass additional props
+
+```jsx
+subscribe_to_newsletter: {
+    title: 'Subscribe to newsletter',
+    type: 'boolean',
+    default: false,
+    labels: ['yes', 'no'],
+},
+```
+
+```{code-block} jsx
+:emphasize-lines: 5,13
+import { Checkbox } from 'semantic-ui-react';
+import FormFieldWrapper from '@plone/volto/components/manage/Widgets/FormFieldWrapper';
+
+export function ToggleWidget(props) {
+  const { id, value, onChange, labels = ['On', 'Off'] } = props;
+  return (
+    <FormFieldWrapper {...props} className="boolean toggle">
+      <Checkbox
+        toggle
+        id={id}
+        checked={value}
+        onChange={(e, { checked }) => onChange(id, checked)}
+        label={value ? labels[0] : labels[1]}
+      />
+    </FormFieldWrapper>
+  );
+}
+
+export default ToggleWidget;
 ```
 
+You can experiment with other widgets included in Volto core under the {file}`/components/manage/Widgets/VocabularyTermsWidget.jsx` directory for more inspiration and possibilities of enhancing your widgets.
```

Comment:
I believe this:

`/components/manage/Widgets/VocabularyTermsWidget.jsx`

should be

`/components/manage/Widgets/`